src/_helpers/currency.js

```diff
@@ -0,0 +1,185 @@
+const geISOCurrencyList = () => {
```

Comment:
move this to `molecules/Amount` directory only. Since right now no one else will use it. we'll move it here when we have more use cases

Reply:
Okay